### PR TITLE
Add Hermes DE

### PR DESCRIPTION
**Pull request by phallobst**
Alexa rank: 8,912

### CODE REVIEW

**Comment by Carlgo11** [comment]
With 81 open PRs maybe you could hold off on new PRs for a while?
It takes time to review all of them and at the moment there’s only two active maintainers.

**Comment by phallobst** [comment]
No pressure, I know that you are all volunteers. I hope nobody was under the impression that I expect the PRs to be processed immediately.
The problem is that it's just too easy to add another website :)

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**
